src/bytesToUuid.js

```diff
@@ -2,36 +2,39 @@
  * Convert array of 16 byte values to UUID string format of the form:
  * XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX
  */
-var byteToHex = [];
-for (var i = 0; i < 256; ++i) {
-  byteToHex[i] = (i + 0x100).toString(16).substr(1);
+const byteToHex = [];
+
+for (let i = 0; i < 256; ++i) {
+  byteToHex.push((i + 0x100).toString(16).substr(1));
 }
 
 function bytesToUuid(buf, offset) {
-  var i = offset || 0;
-  var bth = byteToHex;
+  const i = offset || 0;
```

Comment:
Yes, IE 11 does not support this feature:
http://kangax.github.io/compat-table/es6/#test-default_function_params

Reply:
Since @ctavan says babel compilation should take care of ES6 issues for IE11, and our browser stack tests should catch the issue if not, let's go ahead and make this change.